src/main/java/org/reactome/release/update_stable_ids/Main.java

```diff
@@ -10,26 +10,26 @@
 
 /**
  * This function iterates through all instances, and checks if it has been changed since the previous release.
- * Instances that have been changed have their 'identifierVersion' attribute incremented by 1 to reflect a change. 
+ * Instances that have been changed have their 'identifierVersion' attribute incremented by 1 to reflect a change.
  *
  */
-public class Main 
+public class Main
 {
 	private static final Logger logger = LogManager.getLogger();
-	
+
     public static void main( String[] args ) throws Exception
     {
     	logger.info("Beginning UpdateStableIds step...");
-       
+
     	String pathToConfig = "";
     	if (args.length > 0) {
     		pathToConfig = args[0];
     	} else {
     		pathToConfig = "src/main/resources/config.properties";
     	}
-       
+
 	System.out.println(pathToConfig);
```

Comment:
Could you remove this println? I think it is an artifact. If we want to keep the filepath for the config file, maybe we should log it instead

Reply:
Done: https://github.com/reactome/release-update-stable-ids/pull/2/commits/65c94eea9c319a3198209e0d82281b37924d502c?utf8=%E2%9C%93&diff=split&w=1